repositories/rclcpp.BUILD.bazel

```diff
@@ -125,3 +125,15 @@ ros2_cpp_library(
         "@ros2_rcl_interfaces//:cpp_lifecycle_msgs",
     ],
 )
+
+ros2_cpp_library(
+    name = "rclcpp_components",
+    srcs = glob(["rclcpp_components/src/*.cpp"]),
+    hdrs = glob(["rclcpp_components/include/**/*.hpp"]),
+    includes = ["rclcpp_components/include"],
```

Comment:
Per https://github.com/ros2/rclcpp/blob/16.0.1/rclcpp_components/CMakeLists.txt, I think we need finer specs for this library. https://github.com/ros2/rclcpp/blob/16.0.1/rclcpp_components/CMakeLists.txt#L24 suggested we may only need headers. Can you take a look at this?